packages/arb-bridge-eth/contracts/bridge/Inbox.sol

```diff
@@ -34,6 +34,12 @@ contract Inbox is IInbox {
 
     IBridge public override bridge;
 
+    event DepositEthRetryable(
```

Comment:
Let's skip the Inbox.sol changes for now in the interest of finalizing for the testnet